tests/test_simplex_collection.cpp

```diff
@@ -1233,4 +1233,47 @@ TEST_CASE("simplex_link_condtion_trimesh", "[simplex_collection]")
         REQUIRE(link_condition(m, t2) == false);
         REQUIRE(link_condition(m, t3) == true);
     }
+}
+
+TEST_CASE("are_simplex_collections_equal", "[simplex_collection]")
```

Comment:
The test is not sufficient but it is fine for now. Please write a TODO that this test should be extended to cases where the two sets are not equal. I am very sure that the code works correctly but it would be good to have full coverage of all possible cases.